lib/rubocop/cop/layout/comment_indentation.rb

```diff
@@ -4,6 +4,24 @@ module RuboCop
   module Cop
     module Layout
       # This cops checks the indentation of comments.
+
```

Comment:
Could you remove this empty line and execute `generate_cops_documentation` rake task?
"This cops checks the indentation of comments" disappears from the documentation( https://github.com/bbatsov/rubocop/pull/4915/files#diff-c4a623a000731c5a177ec5b63a7af647L407 ) by this empty line.